### DISCUSSION TITLE
Global Trekker

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#810] [Q&A]
### Channel ID

GlobalTrekker.sg

### Notes

There's a stream here: http://live.tvfix.org/play/smithsonian but the m3u8 link itself doesn't work as is.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19782
